src/osd/OSD.h

```diff
@@ -1406,13 +1406,25 @@ class OSD : public Dispatcher,
   }
 
   static void split_list(
+    boost::intrusive::list<OpRequest> *from,
+    boost::intrusive::list<OpRequest> *to,
+    unsigned match,
+    unsigned bits) {
+    for (auto i = from->begin(); i != from->end(); ) {
+      if (split_request(&(*i), match, bits)) {
+	to->push_back(*i);
```

Comment:
split won't work i think because it's dependent on split_request. this code is broken, though, because an op can only be on one intrusive list at a time; fixing